tests/meshcop/commissioner_argcargv.cpp

```diff
@@ -31,17 +31,22 @@
  *   The file implements the command line params for the commissioner test app.
```

Comment:
Consider using an existing third-party command-line argument parser?

Reply:
We can use getopt, but since all the args have been already well defined before, I tend not to break them.